docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -77,6 +77,14 @@ KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
+* Edit /etc/kubernetes/controller-manager to add fed-node to the
+  KUBELET_ADDRESSES.
+
+```
+# Comma separated list of minions
+KUBELET_ADDRESSES="--machines=fed-node"
```

Comment:
it works, is perfectly valid, and very simple for this 2 node (1 master, 1 worker) example.

pls, when --machines is removed create a simple alternative, maybe kubectl sub-command, method for adding/removing nodes.